frontend/src/locales/sql-review/en-US.json

```diff
@@ -117,6 +117,10 @@
         }
       }
     },
+    "naming-fully-qualified": {
```

Comment:
The title and description are information-less. Users cannot understand the "Fully qualified object name" means.